### DISCUSSION TITLE
Cubesatsim is done! de Virginia NV5F

### DISCUSSION THREAD

**Discussion by Boovtech** [alanbjohnston/CubeSatSim#176] [Show and tell]
Hi all,

I am happy to report that once I figured out that I needed a new USB cable for the Pro Micro, all went well and I finished my simulator late last week.  I tested things as I put it together just to make sure nothing was adversely affected as I put it all together.  It is given to spontaneously rebooting now and then, but it would not be something I built without a little "character."

All modes are functioning and all solar panels are recording voltage.  I will bring it to the Arlington ARC show and tell this week and will  be doing a presentation of it at the Dallas ARC in October.  

I chose to use sockets to put the sensors on the payload board in case one of them decided not to cooperate and might need to be replaced in the future.  I also did something similar to what Mike did, using sockets for the blue INA219 sensors on the main board.  The blue sensors are a little crowded, but only two of them needed little notches Dremmeled out of the edges to clear the standoffs and that was no big deal. In fact, it worked perfectly.

Also, I had to use slightly different narrow panels on the camera side so there's a wire showing that I could not avoid.

Here are a few photos.

Enjoy!

73
Virginia
![20220806_134746](https://user-images.githubusercontent.com/108498218/184799769-e9d3a9b4-64e0-46be-b1a5-b37ca96f0af6.jpg)
![20220808_192238](https://user-images.githubusercontent.com/108498218/184799804-15878198-078f-46ed-9758-4c9a12d32308.jpg)
![20220808_191733_HDR](https://user-images.githubusercontent.com/108498218/184799826-291a27c2-03d5-4639-8223-1ae94bf8738c.jpg)
![20220809_221537](https://user-images.githubusercontent.com/108498218/184799851-95211801-04e2-4b4d-8e89-c432155b357a.jpg)
![20220811_182313](https://user-images.githubusercontent.com/108498218/184799881-a75913e9-16f0-4dd4-b27d-9014c317ed75.jpg)
![20220812_115144](https://user-images.githubusercontent.com/108498218/184799898-f309fa9f-82f4-4d1c-996f-053bd89fb165.jpg)
![20220811_221033](https://user-images.githubusercontent.com/108498218/184799916-23493889-88a7-4be5-a14e-b2d7e7f70ac6.jpg)

**Comment by W7DGP** [comment]
Congratulations! It really looks nice and your data is right on. 
I should finish mine in a couple weeks. I'm at the prepping solar panels stage.

**Reply by Boovtech** [reply]
Thanks so much.  I hope the completion of yours goes well!

**Comment by MikeMcPherson** [comment]
Nice AMSAT Journal article!  Congratulations.  And it was kind of you to give me a shout-out for my truly amazing Dremel skills! :-)

Mike KQ9P

**Reply by Boovtech** [reply]
Thank you Mike!

I did find your solution inspiring.  Up to that point I'd been kind of stuck.

73
Virginia NV5F
